app/src/main/java/com/example/lhi06/residue/AlbumTrackListResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.example.lhi06.residue;
+
+import java.util.List;
+
+/**
+ * Created by lhi06 on 27/07/2017.
+ */
+
+public class AlbumTrackListResponse {
+
+    private List<Track> results;
```

Comment:
Yes, you would need to initialise it to null in the declaration or add a constructor so on second thoughts probably not worth worrying about